src/collection-view.js

```diff
@@ -19,16 +19,16 @@ Marionette.CollectionView = Marionette.View.extend({
   // option to pass `{comparator: compFunction()}` to allow the `CollectionView`
   // to use a custom sort order for the collection.
   constructor: function(options){
-    var initOptions = options || {};
-    if (_.isUndefined(this.sort)){
-      this.sort = _.isUndefined(initOptions.sort) ? true : initOptions.sort;
-    }
 
     this.once('render', this._initialEvents);
     this._initChildViewStorage();
 
     Marionette.View.apply(this, arguments);
 
+    if (_.isUndefined(this.sort)) {
+        _.defaults(this.options, { sort: true });
```

Comment:
I'm not sure?
Originally with it being `this.sort = _.isUndefined(initOptions.sort) ? true : initOptions.sort;`  all I've really changes is `this.sort` -> `this.options.sort` but `_.defaults` seemed more readable.